src/components/pages/rules/flyouts/ruleDelete/deleteRule.scss

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+
+.delete-rule-flyout-container {
+
+  .padded-top-bottom {
+    margin-left: 0px;
+    @include rem-fallback(padding, 24px, 0px);
+  }
+
+  .delete-info {
+    display: flex;
+    .asterisk-svg {
+      align-self: flex-start;
+    }
+  }
+
+  .delete-confirmation-text {
+    display: flex;
+    .check-svg {
+      align-self: flex-start;
+    }
+  }
+
+  .rule-details {
+    border-top-style: none;
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+  }
+
+  .rule-delete-error {
+    border: 1px transparent solid;
+    @include rem-fallback(margin, 12px, 0px);
+    @include rem-fallback(padding, 16px);
+  }
+
+  @include themify($themes) {
+
+    .delete-info a, .delete-confirmation a { color: themed('colorFlyoutConfirmationText'); }
+
+    .delete-info, .delete-confirmation {
+      color: themed('colorFlyoutConfirmationText');
+      @include rem-fallback(font-size, 14px);
```

Comment:
Only property definitions that rely on `themed` should exist inside the `themify`. All other property definitions should occur outside of it.